coderd/database/dbmem/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package dbmem no longer exists.
+// This file exists so that the package can be imported without errors.
+
+package dbmem
```

Comment:
When I tried updating the telemetry server, I ran into this issue:

```
go: github.com/coder/coder-telemetry-server imports
        github.com/coder/coder/v2_12_1/coderd/telemetry tested by
        github.com/coder/coder/v2_12_1/coderd/telemetry.test imports
        github.com/coder/coder/v2/coderd/database/dbmem: module github.com/coder/coder/v2@latest found (v2.25.1, replaced by /Users/cian/src/coder/coder), but does not contain package github.com/coder/coder/v2/coderd/database/dbmem
```

This is my current workaround. I'm leaving it here in the hope that there's a better fix...